src/ripple/protocol/impl/TxFormats.cpp

```diff
@@ -25,6 +25,24 @@ namespace ripple {
 TxFormats::TxFormats()
 {
     // Fields shared by all txFormats:
```

Comment:
We should probably update this comment and explain what the `common` and `pseudo` vars are doing and how they differ.